### PR TITLE
reflect that ExpressionBuilder#andX() and orX() params are variadic in PHPDoc

### PR DESCRIPTION
**Pull request by someniatko**
Like https://github.com/doctrine/collections/pull/217 but changes only PHPDoc

### CODE REVIEW

File: lib/Doctrine/Common/Collections/ExpressionBuilder.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 class ExpressionBuilder
 {
     /**
-     * @param mixed $x
+     * @param mixed ...$x
```
Comment (Ocramius) [inline lib/Doctrine/Common/Collections/ExpressionBuilder.php]: This syntax is not really understood by current tooling 🤔

Comment (someniatko) [reply lib/Doctrine/Common/Collections/ExpressionBuilder.php]: https://github.com/vimeo/psalm/blob/master/tests/VariadicTest.php

Comment (Ocramius) [reply lib/Doctrine/Common/Collections/ExpressionBuilder.php]: Oh, that's very good news!

**Comment by someniatko** [comment]
force-pushed to change commit author

**Comment by alcaeus** [comment]
@Ocramius sometimes GitHub and Travis-ci just need a gentle kick in the right direction ;)

**Pull request merged**